google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -262,11 +262,18 @@ def parse_insert(insert_sql, params):
     if not params:
         # Case a) perhaps?
         # Check if any %s exists.
-        pyformat_str_count = after_values_sql.count("%s")
-        if pyformat_str_count > 0:
-            raise ProgrammingError(
-                'no params yet there are %d "%s" tokens' % pyformat_str_count
-            )
+
+        # pyformat_str_count = after_values_sql.count("%s")
+        # if pyformat_str_count > 0:
+        #     raise ProgrammingError(
+        #         'no params yet there are %d "%%s" tokens' % pyformat_str_count
+        #     )
+        for item in after_values_sql:
+            if item.count("%s") > 0:
+                raise ProgrammingError(
+                    'no params yet there are %d "%%s" tokens'
+                    % item.count("%s")
+                )
```

Comment:
The `after_values_sql` is a list, while `.count()` method applies the search to a string, making it necessary to iterate over the list items.